Jenkinsfile

```diff
@@ -75,7 +75,7 @@ node( 'hcctest' )
   }
 
 // JENKINS-33510: the jenkinsfile dir() command is not workin well with docker.inside()
-  hcc_build_image.inside( '--privileged --device=/dev/kfd --device=/dev/dri --group-add video' )
+  hcc_build_image.inside( '--privileged' )
```

Comment:
can we remove `--privileged` and keep the other flags instead?

Reply:
yup.  trying that now.